package/MDAnalysis/analysis/encore/clustering/src/ap.c

```diff
@@ -43,7 +43,7 @@ float min(float * values, int length) { //array min
 	return min;
 }
 
-float max(float * values, int length) { //array max
+float maxx(float * values, int length) { //array max
```

Comment:
are the functions still used on the linux build now?

Reply:
What this `max` thing in encore is, changing its name to `maxx` allows the CI suite to pass for linux so I will likely wrap code involving ` max` in a preprocessor directive that does not allow it to be declared / used on Windows (because `max` actually blocks windows build / compile). If it were important it would cause unit test failure to rename like this.

Reasonable?